src/Misc/layoutbin/en-US/strings.json

```diff
@@ -157,6 +157,8 @@
     "--auth                 Authentication type. Valid options are PAT (Personal Access Token),",
     "                       Negotiate (Kerberos or NTLM), Integrated (Windows default credentials) and",
     "                       ALT (Alternate Credentials)",
+    "--work                 Agent work folder, default to _work folder under agent root,",
```

Comment:
@bryanmacfarlane if we do stamp the work dir, could we please consider dockerised deployments please. In our case we keep work dir on the host and map it into the running container. The containers stop and start but working directory stays the same. If the stamping gets introduced, lets hope this does not cause issues in this scenario?